packages/old-accounter/src/data/save-transactions-to-db.ts

```diff
@@ -69,13 +71,13 @@ export async function saveTransactionsToDB<
       // }
     }
 
-    let tableName = `poalim_${accountType}_account_transactions`;
-    if (accountType == 'isracard') {
-      tableName = 'isracard_creditcard_transactions';
-    }
-    if (accountType == 'amex') {
-      tableName = 'amex_creditcard_transactions';
+    function getTableName(accountType: AccountTypes): string {
+      if (accountType === 'isracard') return 'isracard_creditcard_transactions';
```

Comment:
Is there a check for if a transaction already exists in the DB?

Reply:
This code was just a cleanup of what existed.

I'll double check I'm not adding in duplicate transactions

Reply:
I checked again:
The insert will fail if a duplicate entry is added.

Reply:
based on what fields does it decides if it was existing or not?